Specs/addDefaultMatchers.js

```diff
@@ -416,6 +432,17 @@ define([
         };
     }
 
+    // converts x and y coordinates such that after updating for drawing buffer position, the correct coordinates will be used
+    function pickPrimitiveEqualsWrapper(actual, expected, x, y, width, height) {
+        width = width || 3;
+        height = height || width || 3;
+        x = x || 0;
+        y = y || 0;
+        var adjustedX = x + ((width - 1) * 0.5);
+        var adjustedY = -1 * (y + ((height - 1) * 0.5) - actual._context.drawingBufferHeight);
+        return pickPrimitiveEquals(actual, expected, adjustedX, adjustedY, width, height);
```

Comment:
this conversion is the inverse of 
```
scratchRectangle.x = drawingBufferPosition.x - ((rectangleWidth - 1.0) * 0.5);
scratchRectangle.y = (this.drawingBufferHeight - drawingBufferPosition.y) - ((rectangleHeight - 1.0) * 0.5);
```
from lines 2829 in Scene.js

Honestly, I don't understand what this logic is accomplishing, but for the test I need to set the bounding box for the pick search to be centered at a position exactly far enough from the rectangle for the increase in pick size to affect the outcome. This was just something that would never work right until I implemented this logic.